tests/router_unit_test.py

```diff
@@ -102,3 +102,29 @@ def test__uses_the_toggle_context_when_it_is_provided(
         toggle_router.get_toggles(["some_toggle"], toggle_context)
 
         toggle_strategy.is_enabled.assert_called_with(context=toggle_context)
+
+    @pytest.mark.parametrize("environment_name", [("production"), ("stage")])
+    def test__get_all_toggles(
+        self,
+        monkeypatch,
+        add_toggles: typing.Callable[[typing.Dict[str, bool]], None],
+        environment_name: str,
+    ):
+        monkeypatch.setenv("ENVIRONMENT", environment_name)
+        toggles = {
+            environment_name: {
+                "some_toggle": {"enabled": True},
```

Comment:
One thing that got me thinking is what will happen with the non-static toggles, for example a pilot users toggle, that uses a toggle context